op-batcher/batcher/driver.go

```diff
@@ -105,7 +105,7 @@ type BatchSubmitter struct {
 	killCtx           context.Context
 	cancelKillCtx     context.CancelFunc
 
-	l2BlockAdded chan struct{} // notifies the throttling loop whenever an l2 block is added
+	pendingBytesUpdated chan int64 // notifies the throttling with the new pending bytes
```

Comment:
This was upgraded from sending empty signal to sending the actual information the throttling loop needs to do its job. This allows the throttling loop to not require the channel manager mutex, since it won't read from there anymore.